lib/bisu.rb

```diff
@@ -33,6 +33,9 @@ def run(options)
     end
 
     Bisu::Logger.print_summary
+    if (options[:strict] && Bisu::Logger.summary[:warn] > 0)
```

Comment:
[minor] your `(...)` brackets here are not needed :)

Reply:
Removed the brackets!